extensions/wikia/LookupContribs/templates/main-form.tmpl.php

```diff
@@ -2,8 +2,8 @@
 <!-- DISTRIBUTION TABLE -->
 <script type="text/javascript" charset="utf-8">
 function lcShowDetails(dbname, mode) {
-	var username = '<?= urlencode( $username ) ?>';
-	var action = '<?=$action?>';
+	var username = '<?= Sanitizer::cleanUrl( $username ) ?>';
+	var action = '<?= Sanitizer::cleanUrl( $action ) ?>';
```

Comment:
Both of these should be `Xml::escapeJsString`. `Sanitizer::cleanUrl` actually unescapes HTML entities... The username should then be urlencoded when it is being used in the URL with `encodeURIComponent` (or just wrap it in `encodeURIComponent` here).